src/devices_models/devices/common/timeseries_constraint.jl

```diff
@@ -114,23 +145,46 @@ Constructs upper bound for given variable using a parameter. The constraint is
 """
 function device_timeseries_param_ub(psi_container::PSIContainer,
```

Comment:
Same here, we need to create a ts_data object that contains everything, there shouldn't be changes here either.